### PR TITLE
Upgrade uniffi

### PR DESCRIPTION
**Pull request by julioromano**
This will include https://github.com/mozilla/uniffi-rs/pull/1781
Which fixes https://github.com/mozilla/uniffi-rs/issues/1760

### CODE REVIEW

File: bindings/matrix-sdk-ffi/src/timeline.rs [hunk 1]
```diff
@@ -832,10 +832,7 @@ impl From<RumaUnstableAudioDetailsContentBlock> for UnstableAudioDetailsContent
 }
 
 #[derive(Clone, uniffi::Record)]
-pub struct UnstableVoiceContent {
-    // DO NOT USE: Dummy field to work around https://github.com/mozilla/uniffi-rs/issues/1760
-    do_not_use: bool,
-}
+pub struct UnstableVoiceContent
```
Comment (jplatte) [inline bindings/matrix-sdk-ffi/src/timeline.rs:835]: You still need the `{}` 😄

Comment (julioromano) [reply bindings/matrix-sdk-ffi/src/timeline.rs:835]: I told you my Rust got rusty :(

Comment (jplatte) [reply bindings/matrix-sdk-ffi/src/timeline.rs]: No worries!

**Comment by julioromano** [comment]
After manual testing I can confirm it builds and runs on EXA.
The generated code for the `UnstableVoiceContent` empty struct is as expected:

```
package org.matrix.rustcomponents.sdk

public final class UnstableVoiceContent public constructor() {
    public companion object {
    }

    public open operator fun equals(other: kotlin.Any?): kotlin.Boolean { /* compiled code */ }

    public open fun hashCode(): kotlin.Int { /* compiled code */ }
}
```

**Review by jplatte — APPROVED** [review]

**Pull request merged**